apps/storybook/.storybook/preview.js

```diff
@@ -0,0 +1,14 @@
+import React, { Fragment } from 'react';
+import { addDecorator } from '@storybook/react';
+import { boolean } from '@storybook/addon-knobs';
+import { ThemeProvider, HtmlDirection } from '@lundium/theme-basic';
```

Comment:
ThemeProvider and HtmlDirection should be part of the core library. the theme-* packages provides just different variables and themes.